smart_home/__init__.py

```diff
@@ -44,6 +45,12 @@ def todayStamps():
     today = int(time.mktime(time.strptime(today,"%Y-%m-%d")))
     return today, today+3600*24
 
+def apiCall():
```

Comment:
This function is only for debugging purpose, I don't think it should be included in this PR